src/awkward1/__init__.py

```diff
@@ -63,7 +63,6 @@
 from awkward1.operations.describe import *
 from awkward1.operations.structure import *
 from awkward1.operations.reducers import *
-from awkward1.operations.transfers import *
```

Comment:
I moved the `copy_to` function into convert.py and renamed it `to_kernels` to fit the pattern of the other "from" and "to" functions.